internal/services/authorization/role_management_policy_resource_test.go

```diff
@@ -343,26 +453,58 @@ resource "azurerm_role_management_policy" "test" {
   }
 
   notification_rules {
+    active_assignments {
+      approver_notifications {
+        notification_level    = "Critical"
+        default_recipients    = false
+        additional_recipients = ["resource-group-approver@example.com"]
+      }
+      # admin_notifications {
+      #   notification_level    = "Critical"
+      #   default_recipients    = false
+      #   additional_recipients = ["resource-group-admin@example.com"]
+      # }
+      # assignee_notifications {
+      #   notification_level    = "All"
+      #   default_recipients    = true
+      #   additional_recipients = ["resource-group-assignee.else@example.com"]
+      # }
```

Comment:
i added these commented-out lines of config to show more clearly what was changing between the steps